src/plugins/workspace/public/plugin.ts

```diff
@@ -372,7 +381,43 @@ export class WorkspacePlugin implements Plugin<{}, {}, WorkspacePluginSetupDeps>
     return {};
   }
 
-  public start(core: CoreStart) {
+  private registerGetStartedCardToNewHome(
+    core: CoreStart,
+    contentManagement: ContentManagementPluginStart
+  ) {
+    // console.log('------');
+    // debugger;
```

Comment:
Seems the comment out lines are not removed.

Reply:
Updated